### PR TITLE
Move installation script to a separate file

### PR DESCRIPTION
**Pull request by kichalla**
- [ ] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [ ] Tests are included and/or updated for code changes.
- [ ] Proper license headers are included in each file.

### CODE REVIEW

File: images/build/Dockerfiles/ltsVersions.Dockerfile [hunk 1]
```diff
@@ -171,6 +174,8 @@ RUN set -ex \
         libonig-dev \
     && rm -rf /var/lib/apt/lists/* \
     && cp -f $imagesDir/build/benv.sh /opt/oryx/benv \
+    && cp -f $imagesDir/installPlatform.sh /opt/oryx/installPlatform \
```
Comment (qianz2) [inline images/build/Dockerfiles/ltsVersions.Dockerfile:177]: So installPlatform will act like a command same as benv in the container? Are we able to run command like 'installPlatform -p python -v 3.9' inside the image?

Comment (kichalla) [reply images/build/Dockerfiles/ltsVersions.Dockerfile:177]: `installPlatform` will be used to install platform versions either when building the image in our CI or even when dynamically installing platforms during customer app's build time too. Once dynamic install is approved for runtime images, we can use this script to install dynamically in runtime images too

**Comment by arroyc** [comment]
this concept is scalable which is good, only thing is we may break things  here and there because this is mostly changes in the bash script  ... I would trigger a nightly build to make sure eveything is ok

File: images/build/Dockerfiles/Dockerfile [hunk 2]
```diff
@@ -47,21 +47,24 @@ RUN set -ex \
...
-    && $imagesDir/installPlatform.sh nodejs 8.12.0 \
-    && $imagesDir/installPlatform.sh nodejs 8.15.1 \
-    && $imagesDir/installPlatform.sh nodejs 9.4.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.1.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.10.0 \
-    && $imagesDir/installPlatform.sh nodejs 10.14.2 \
-    && $imagesDir/installPlatform.sh nodejs 10.21.0 \
-    && $imagesDir/installPlatform.sh nodejs 12.18.2 \
+    && . $buildDir/__sdkStorageConstants.sh \
+    && export DYNAMIC_INSTALL_ROOT_DIR="/opt" \
```
Comment (arroyc) [inline images/build/Dockerfiles/Dockerfile:51]: just curious .. aren't we using /tmp for this location? why choosing /opt?

Comment (kichalla) [reply images/build/Dockerfiles/Dockerfile:51]: This is for baked in platforms in the image which we install under `/opt`

**Pull request closed without merging**